src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,31 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected, setActiveTabId }) => {
```

Comment:
The `onTabSelected` prop is expected to be a callback function, but it is being used as a tab object in the `App.jsx` file. You should update the `App.jsx` to pass a function that handles tab selection logic.